src/api/users/users.router.js

```diff
@@ -4,5 +4,6 @@ import * as usersController from './users.controller.js';
 const router = Router();
 
 router.get('/:id', usersController.getById);
+router.get('/byEmail', usersController.getByEmail);
```

Comment:
In usersController.getByEmail, the user's email address is taken from params, but here you don't have ":email" (for example) after your path...

Reply:
for next commit the path is now changed to /byEmail/:email